File: CarlaErik/UvA.SoftCon.Questionnaire/AST/Expressions/Boolean/AndExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UvA.SoftCon.Questionnaire.AST.Expressions.Boolean
{
    public class AndExpression : BinaryBooleanExpression<IBooleanExpression, IBooleanExpression>
    {
        public AndExpression(IBooleanExpression left, IBooleanExpression right)
            : base(left, right) { }

        public override bool Evaluate()
        {
            return Left.Evaluate() && Right.Evaluate();
        }
    }
}

File: KrasiMilan/QL/src/main/java/nl/uva/softwcons/ast/expression/binary/logical/AndExpression.java
package nl.uva.softwcons.ast.expression.binary.logical;

import nl.uva.softwcons.ast.expression.Expression;
import nl.uva.softwcons.ast.expression.binary.BinaryExpression;

public class AndExpression extends BinaryExpression {

    public AndExpression(Expression left, Expression right) {
        super(left, right);
    }

}
